### DISCUSSION TITLE
any plan to add 22h2 support for windows 11

### DISCUSSION THREAD

**Discussion by yingyu-wang** [AveYo/MediaCreationTool.bat#156] [Q&A] [answered]
22h2 support integration for windows 11

**Comment by AveYo** [comment, marked as answer]
Obviously ;) I had it ready since the esd's were published by Microsoft, 
but I want to wait a bit to see if they release a svc for it _(it's not a great build to begin 22H2 with)_.
And I also want to redesign the whole thing to add some kick-ass features. Stay tuned.

**Reply by MarceloWofman** [reply]
Consulta ... existe alguna posibilidad de tenerla 22H2 y si ahy mejoras luego incorporalas o sea ir actualixando... MIL GRACIAS  POR TODO TU A PORTE ... relamete poder tener el AutoUpdate22H2 seria GENIALLL !!!

**Reply by MarceloWofman** [reply]
PELASE AutoUpdate22H2 FOR WIN 11 PRO

**Comment by joel-schaal** [comment]
Hello @AveYo !

I just discovered this insanely ingenious script and wanted to thank you for that.
Provisioning ephemeral Windows VMs for local tests is a bane.

I wanted to know if there was any news concerning that update ? 
Would you make this available for Windows 10 too ?

Have a great day :)
